src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+
+Cypress.Commands.add('checks', () => {
+  cy.axeCheck();
+
+  // Search
+  cy.get('#street-city-state-zip').type('Austin, TX');
+  cy.get('#facility-type-dropdown').select('VA health');
+  cy.get('#facility-search').click();
```

Comment:
Maybe add a test that confirms the serviceType field behavior?

- disabled when non-community is selected
- enabled when community selected